tests/unit/utils/elastic_search/transformations/test_tx.py

```diff
@@ -0,0 +1,8 @@
+from tests.fixtures.elastic_search_fixtures import ELASTIC_SEARCH_RAW_HIT, ES_HIT_TO_RESULT_EXPECTED_RETURN
+from utils.elastic_search.transformations.tx import es_hit_to_result
+
+
+class TestTx:
+    def test_tx_transformation(self):
```

Comment:
Is there no "error" case?
For instance if the es hit cannot be transformed because it's missing some property.

Reply:
I will add a new `except`on `usecases/block_api` to inform client that document is missing information